### DISCUSSION TITLE
Enabling language switcher

### DISCUSSION THREAD

**Discussion by vanillaiice** [binokochumolvarghese/lightbi-hugo#70] [Q&A] [answered]
Hi. First of all thank you for this amazing theme. I am trying to set up multiple languages for my blog. I followed the guide and set up the hugo config and the different content directories (namely, content/en  content/fr). 

Everything is working fine, aside for one thing, I can't seem to find a way to enable a language switcher/selector.

Any clues on how to do that ?

**Comment by binokochumolvarghese** [comment, marked as answer]
I am sorry as of now there is not language switcher. I will look into that.

**Comment by binokochumolvarghese** [comment]
I have updated language switcher in the menu. Please use the latest version of the theme.
